### PR TITLE
Bring back risk summary section

### PR DESCRIPTION
**Pull request by maciekzygmunt**
Resolves L2B-4542

### CODE REVIEW

**Comment by torztomasz** [comment]
Do we want to repeat this warning? 

image
image

File: packages/frontend/src/pages/layer3s/props/getProjectDetails.ts [hunk 1]
```diff
@@ -118,6 +120,14 @@ export function getProjectDetails(
       },
     })
 
+    const riskSection = getRiskSection(project, verificationStatus)
```
Comment (torztomasz) [inline packages/frontend/src/pages/layer3s/props/getProjectDetails.ts:123]: lets put getters in the top of this function

File: packages/frontend/src/pages/scaling/projects/props/getProjectDetails.ts [hunk 2]
```diff
@@ -135,6 +137,14 @@ export function getProjectDetails(
       })
     }
 
+    const riskSection = getRiskSection(project, verificationStatus)
```
Comment (torztomasz) [inline packages/frontend/src/pages/scaling/projects/props/getProjectDetails.ts:140]: same

File: packages/frontend/src/utils/project/getRiskSection.ts [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+  return {
+    id: 'risk-summary',
+    title: 'Risk summary',
+    riskGroups: groupRisks(risks),
+    warning: project.display.warning,
+    isVerified: verificationStatus.projects[project.id.toString()],
+    redWarning: undefined,
+  }
+}
```
Comment (torztomasz) [inline packages/frontend/src/utils/project/getRiskSection.ts:62]: Where did you get this fn? Is this some old one that was used?

**Review by torztomasz — CHANGES_REQUESTED** [review]
Do we want to repeat that warning?

File: packages/frontend/src/pages/scaling/projects/props/getProjectDetails.ts (hunk 2 above)
Comment (maciekzygmunt) [reply packages/frontend/src/pages/scaling/projects/props/getProjectDetails.ts]: [6e2457b](https://github.com/l2beat/l2beat/pull/3360/commits/6e2457b0e61c48b19486b2abd9c46711dd18dfc0)

File: packages/frontend/src/pages/layer3s/props/getProjectDetails.ts (hunk 1 above)
Comment (maciekzygmunt) [reply packages/frontend/src/pages/layer3s/props/getProjectDetails.ts]: [6e2457b](https://github.com/l2beat/l2beat/pull/3360/commits/6e2457b0e61c48b19486b2abd9c46711dd18dfc0)

**Review by torztomasz — APPROVED** [review]

**Pull request merged**